scripts/quests/jeuno/LB09_1_Dormant_Powers_Dislodged.lua

```diff
@@ -0,0 +1,151 @@
+                    end
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest accepted.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(10192) -- Timing Minigame starting event.
+                    else
+                        local itemWanted = quest:getVar(player, 'itemWanted') - 1
+                        return quest:event(10045, 0, 1, 4, 1, 0, itemWanted)
+                    end
+                end,
+
+                onTrade = function(player, npc, trade)
+                    local itemWanted  = quest:getVar(player, 'itemWanted') - 1
+                    local itemToTrade = itemWantedTable[itemWanted][1]
+
+                    if npcUtil.tradeHasExactly(trade, {{xi.items.KINDREDS_CREST, 1}, {itemToTrade, 1}}) and
+                        player:getMeritCount() > 9
+                    then
+                        return quest:progressEvent(10191)
+                    end
+                end,
+            },
+
+            onEventUpdate =
+            {
+                [10192] = function(player, csid, option, npc)
+                    -- Option: Push! The option is the number of frames. 1 sec = 60 frames. Probably.
+                    if option >= 901 then
+                        player:updateEvent(4, 1) -- Slime. Way too late?
+                    elseif option >= 631 and option <= 900 then
+                        player:updateEvent(3, 1) -- Big Egg. Too late?
+                    elseif option >= 571 and option <= 630 then
+                        player:updateEvent(2, 1) -- Glowing Egg. Win!
+                        quest:setVar(player, 'Win', 1)
+                    elseif option >= 301 and option <= 570 then
+                        player:updateEvent(1, 1) -- Regular Egg. Too soon?
+                    elseif option <= 360 then
+                        player:updateEvent(0, 1) -- Mandragora. Way too soon?
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10191] = function(player, csid, option, npc)
+                    player:tradeComplete()
+                    player:setMerits(player:getMeritCount() - 10)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+
+                [10192] = function(player, csid, option, npc)
+                    if quest:getVar(player, 'Win') == 1 then
+                        if quest:complete(player) then
+                            player:setLevelCap(95)
+                            player:messageSpecial(ID.text.YOUR_LEVEL_LIMIT_IS_NOW_95)
+                            player:messageSpecial(ID.text.KEYITEM_OBTAINED, xi.ki.SOUL_GEM)
```

Comment:
No need for this messageSpecial if keyItem is in the reward table

Reply:
For some reason, if it isnt there, the message wont show up after the event is finished.

Reply:
Since there's no item reward, just out of curiosity could you try:
setLevelCap
Level limit 95 message
quest:complete(player)
?